src/main/kotlin/com/ampnet/headlesscmsservice/service/pojo/MailResponse.kt

```diff
@@ -0,0 +1,42 @@
+package com.ampnet.headlesscmsservice.service.pojo
+
+import com.ampnet.headlesscmsservice.enums.Lang
+import com.ampnet.headlesscmsservice.enums.MailType
+import com.ampnet.headlesscmsservice.persistence.model.Mail
+import com.ampnet.headlesscmsservice.persistence.model.MailId
+
+data class MailResponse(
+    val id: Int,
+    val coop: String,
+    val title: String,
+    val content: String,
+    val type: MailType,
+    val requiredFields: List<String>,
+    val lang: Lang
+) {
+    constructor(mail: Mail) : this(
+        mail.id,
+        mail.coop,
+        mail.title,
+        mail.content,
+        mail.type,
+        mail.type.getRequiredFields().map { it.value },
+        mail.lang,
+    )
+
+    constructor(
+        coop: String,
+        title: String,
+        content: String,
+        type: MailType,
+        lang: Lang
+    ) : this(
+        MailId(coop, type, lang).hashCode(),
```

Comment:
This will return mail id which is not stored in the database. How about setting `id: Int?` and return `null` if the mail is not stored in db? `id: null` will also indicate to the frontend that this mail has a default value.

Reply:
Good suggestion, I'll implement it.